pkg/environment/history.go

```diff
@@ -245,17 +245,29 @@ func getLatestHistoryFile(dir string) (item historyItem) {
 
 // HidePassword replace password with ******
 func HidePassword(args []string) []string {
-	var record []string
+	redactArgs := []string{
+		"-p",
+		"--tidb-password",
```

Comment:
Do we have any other possible password arguments?

Reply:
Added another for dumpling, and comments, PTAL